submissions/drewzag/memory-pair-game/app.js

```diff
@@ -0,0 +1,87 @@
+const section = document.querySelector('section')
+
+const getData = () => [
+  { imgSrc: './images/ari.webp', id: 'arisona' },
+  { imgSrc: './images/cle.webp', id: 'clevlend' },
+  { imgSrc: './images/lv.webp', id: 'las-vegas' },
+  { imgSrc: './images/mia.webp', id: 'miami' },
+  { imgSrc: './images/sea.webp', id: 'seattle' },
+  { imgSrc: './images/tb.webp', id: 'tampa-bay' },
+]
+
+const randomize = () => {
+  const cardData = getData().concat(getData())
+  cardData.sort(() => Math.random() - 0.5)
+  return cardData
+}
+
+const cardGenerator = () => {
+  const cardData = randomize()
+
+  const cardsContainer = document.createDocumentFragment()
+  cardData.forEach((item) => {
+    const card = document.createElement('div')
+    const face = document.createElement('img')
+    const back = document.createElement('div')
+    card.classList = 'card'
+    card.setAttribute('name', item.id)
+    face.classList = 'face'
+    back.classList = 'back'
+
+    face.src = item.imgSrc
+
+    card.append(face, back)
+    cardsContainer.appendChild(card)
+
+    card.addEventListener('click', ({ target }) => {
+      card.classList.toggle('toggle')
+      checkCards(target)
+    })
+  })
+  section.appendChild(cardsContainer)
+}
+
+const checkCards = (target) => {
```

Comment:
you can name argument clickedCards, so no need to do assignment  `const clickedCards = target`